app/views/static/index.html.erb

```diff
@@ -26,7 +26,7 @@
         </button>
       </section>
 
-      <% if @current_user&.activities&.any? %>
+      <% if @has_activities %>
```

Comment:
if we move everything to the controller, should prob do the same for https://github.com/tomekr/jamroulette/blob/01df8482afc350bbe9ea4a0ba1cc78a633d47cd9/app/views/static/index.html.erb#L12 too